knowledge-base/docker.md

```diff
@@ -1,103 +1,111 @@
 ## Docker and Docker Compose installation manual
 
-Source website:
-	https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/
+Source website: https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/
```

Comment:
Divide to MacOS and Linux? You're linking to Ubuntu directly.

Reply:
That's just a move of an already existing resource. We can improve it but that's beyond this pull request.